src/index.ts

```diff
@@ -140,16 +141,18 @@ async function startApp() {
       // Supply userAgent for analytics
       res.locals.userAgent = userAgent
 
+      const rootValue: RootValue = {
+        accessToken,
+        userID,
+        defaultTimezone,
+        exchangeSchema,
+        ...loaders,
+      }
```

Comment:
I extracted this so I could ensure that if someone adds a new property then it will fail tsc without it getting added to `types/graphql.d.ts`